amundsen_application/static/js/components/TableDetail/OpenRequestDescription/styles.scss

```diff
@@ -0,0 +1,11 @@
+@import 'variables';
+
+.request-description{
+  display: flex;
+  font-size: 16px;
+  text-decoration: none;
+}
+
+.request-description:hover, .request-description:visited, .request-description:active, .request-description:link {
```

Comment:
formatting nit: Usually each item declaration is on a separate line for easier readability